src/main/pages/che-7/overview/assembly_hosted-che.adoc

```diff
@@ -8,33 +8,56 @@ folder: che-7/overview
 summary: 
 ---
 
-:parent-context-of-hosted-che: {context}
-
-[id='hosted-che_{context}']
+[id="hosted-che"]
 = Hosted Che
 
-:context: hosted-che
+"Hosted Che" is Eclipse Che hosted by Red Hat. Che server is running on OpenShift Dedicated and user base is spread across multiple OpenShift Online clusters where workspaces are created.
+
+[id="registration"]
+== Registration Process
+
+image::hosted-che/registration-1.png[]
+image::hosted-che/registration-2.png[]
+image::hosted-che/registration-3.png[]
+image::hosted-che/registration-4.png[]
+image::hosted-che/registration-5.png[]
+image::hosted-che/registration-6.png[]
+image::hosted-che/registration-7.png[]
+image::hosted-che/registration-8.png[]
+image::hosted-che/registration-9.png[]
+
+Registration Process
+
+[id='github-account-linking']
+== GitHub Account Linking
 
+GitHub Account Linking
```

Comment:
```suggestion
If you need to access code on GitHub, it is recommended to link your GitHub account to your Red Hat account.

. Navigate to https://developers.redhat.com/auth/realms/rhd/account/identity
. Click on the *Link* button next to the GitHub logo.

```

Reply:
Good point, but I believe we would like to promote account linking via Che User Dashboard. This way of linking could be treated as optional